src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -506,6 +491,55 @@ private static StringBuilder fakeBody(final Request... dispositions)
                 .append(String.format("--%s--", BOUNDARY));
             return builder;
         }
+
+        /**
+         * This class is using a decorator pattern for representing
+         * a one-to many header-body relation.
```

Comment:
@sebing can you replace this Javadoc with just `A fake HTTP multipart request.`? I'm not sure if it really represents  `one-to many header-body relation`.